### ISSUE TITLE
Where can I download the official LuaSocket 2.0?

### ISSUE THREAD

**Issue by gundamC** [lunarmodules/luasocket#432]
Where can I download the official LuaSocket 2.0? It is the version of LuaSocket that perfectly supports Lua 5.0.

**Comment by catwell** [comment]
AFAIK no official release of LuaSocket ever worked with Lua 5.0 without compat-5.1, the switch to the 5.1-style module interface was made during the beta.

If you mean with compat-5.1 you can use 2.0.2, there is a corresponding tag on this repository.

May I ask why you are stuck with 5.0 though? 5.1 and the last version of 5.0 were both released 18 years ago, it is probably time to upgrade...

**Comment by gundamC** [comment]
@catwell Thank you for the reply. I found a version that supports Lua 5.0 at this address: https://web.tecgraf.puc-rio.br/luasocket/old/luasocket-2.0-alpha. However, I am not sure if it is an official release. The reason you are curious why I am still using Lua 5.0 is because I found an interesting open source project on GitHub, which is written in Lua 5.0. I am thinking of adding some network functionality to it.

**Comment by Tieske** [comment]
depending on the "interesting project" it might be easier to port that to a newer version of Lua?

**Comment by catwell** [comment]
@gundamC Yes by "official" I meant "final", I think this alpha release does still support Lua 5.0.

I agree with Tieske though, it would maybe be easier to port it to at least Lua 5.1.
